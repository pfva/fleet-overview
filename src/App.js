import Header from './components/UI/Header/Header';
import VehicleTable from './components/VehicleTable/VehicleTable/VehicleTable';
import VehicleProvider from './store/vehicles/VehicleProvider';
import EquipmentsProvider from './store/equipments/EquipmentsProvider';

function App() {
  return (
    <VehicleProvider>
      <EquipmentsProvider>
        <main>
          <Header />
          <VehicleTable />
        </main>
      </EquipmentsProvider>
    </VehicleProvider>
  );
}

export default App;
