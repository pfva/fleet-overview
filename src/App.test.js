import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import App from './App';

describe('App', () => {
  it('renders without crashing', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});