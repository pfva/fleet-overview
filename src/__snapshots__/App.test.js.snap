// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<main>
  <header
    className="header"
  >
    <span
      className="vehicle"
      data-testid="Icon-vehicle"
    >
      <svg
        className="vehicle"
      >
        truck.svg
      </svg>
    </span>
    <div
      className="textWrapper"
    >
      <h1
        className="title"
      >
        Welcome, Fleet Owner
      </h1>
      <p
        className="infoText"
      >
        Here you can view a detailed list of your vehicle fleet.
      </p>
      <p
        className="infoText"
      >
        To edit a vehicle, simply click on the field you want to change.
      </p>
      <p
        className="infoText"
      >
        The updated vehicle will be saved when you press Enter, or click anywhere outside.
      </p>
    </div>
  </header>
  <div
    className="vehicleTable"
  >
    <div
      className="vehicleTableHeader"
    >
      <div
        className="vehicleHeaderWrapper"
      >
        <p
          className="vehicleHeaderCell"
          onClick={[Function]}
        >
          ID
        </p>
      </div>
      <div
        className="vehicleHeaderWrapper"
      >
        <p
          className="vehicleHeaderCell"
          onClick={[Function]}
        >
          Name
        </p>
      </div>
      <div
        className="vehicleHeaderWrapper"
      >
        <p
          className="vehicleHeaderCell"
          onClick={[Function]}
        >
          Driver
        </p>
      </div>
      <div
        className="vehicleHeaderWrapper"
      >
        <p
          className="vehicleHeaderCell"
          onClick={[Function]}
        >
          Status
        </p>
      </div>
      <div
        className="vehicleHeaderWrapper"
      >
        <p
          className="vehicleHeaderCell"
          onClick={[Function]}
        >
          Fuel type
        </p>
      </div>
      <div
        className="vehicleHeaderWrapper"
      >
        <p
          className="vehicleHeaderCell"
          onClick={[Function]}
        >
          Equipments
        </p>
      </div>
    </div>
    <div
      className="vehicleTableRowWrapper"
    >
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v1
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            AB23
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            SpongeBob SquarePants
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            LNG
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          >
            <p
              className="vehicleTableCell"
            >
              Crane
            </p>
            <p
              className="vehicleTableCell"
            >
              Tachograph
            </p>
          </span>
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v2
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            XXW123
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Patrick Star
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Diesel
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          >
            <p
              className="vehicleTableCell"
            >
              Tachograph
            </p>
          </span>
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v3
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            GA33
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Squidward Tentacles
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            CNG
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          >
            <p
              className="vehicleTableCell"
            >
              Hook
            </p>
          </span>
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v4
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            LK21
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Mr. Krabs
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-off"
          onClick={[Function]}
        >
          <span
            className="dot inactive"
            data-testid="Icon-inactive"
          />
          <p>
            <span
              className="dataText"
            >
              inactive
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Diesel
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          />
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v5
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            GHB 5
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Plankton and Karen
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Diesel
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          >
            <p
              className="vehicleTableCell"
            >
              Crane
            </p>
            <p
              className="vehicleTableCell"
            >
              Hook
            </p>
          </span>
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v6
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            LLO21
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Sandy Cheeks
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Diesel
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          >
            <p
              className="vehicleTableCell"
            >
              Fire Extinguisher
            </p>
          </span>
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v7
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            OWL21
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Mrs. Puff
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Electrical
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          />
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v8
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            JNWJ12
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Pearl Krabs
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Eletrical
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          >
            <p
              className="vehicleTableCell"
            >
              Fire Extinguisher
            </p>
          </span>
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v9
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            KL2OK1
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Gary the Snail
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            LNG
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          />
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v10
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            LSAO21
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Patchy the Pirate
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            LNG
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          >
            <p
              className="vehicleTableCell"
            >
              Tachograph
            </p>
          </span>
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v11
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            LSP21P
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Potty the Parrot
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-off"
          onClick={[Function]}
        >
          <span
            className="dot inactive"
            data-testid="Icon-inactive"
          />
          <p>
            <span
              className="dataText"
            >
              inactive
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Diesel
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          >
            <p
              className="vehicleTableCell"
            >
              Hook
            </p>
          </span>
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v12
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            212AWN
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            French Narrator
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Diesel
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          />
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v13
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            1W321W
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Mermaid Man and Barnacle Boy
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Diesel
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          >
            <p
              className="vehicleTableCell"
            >
              Tachograph
            </p>
          </span>
        </div>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v14
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            14HUEO
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            The Flying Dutchman
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Diesel
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          />
        </p>
      </div>
      <div
        className="vehicleTableRow"
        data-testid="VehicleTableRow"
      >
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            v15
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            
          </span>
        </p>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            King Neptune
          </span>
        </p>
        <div
          className="vehicleTableCell iconWrapper"
          data-testid="VehicleStatus-on"
          onClick={[Function]}
        >
          <span
            className="dot active"
            data-testid="Icon-active"
          />
          <p>
            <span
              className="dataText"
            >
              active
            </span>
          </p>
        </div>
        <p
          className="vehicleTableCell"
          data-testid="VehicleTableCell-text"
          onClick={[Function]}
        >
          <span
            className="dataText"
          >
            Diesel
          </span>
        </p>
        <div
          className="vehicleEquipments"
          data-testid="VehicleEquipments-wrapper"
          onClick={[Function]}
        >
          <span
            className="dataText adjustHovered"
          >
            <p
              className="vehicleTableCell"
            >
              Crane
            </p>
            <p
              className="vehicleTableCell"
            >
              Tachograph
            </p>
            <p
              className="vehicleTableCell"
            >
              Fire Extinguisher
            </p>
            <p
              className="vehicleTableCell"
            >
              Hook
            </p>
          </span>
        </div>
      </div>
    </div>
  </div>
</main>
`;
